packages/domain-lib/src/types/infrastructure.ts

```diff
@@ -95,6 +98,8 @@ export interface ISettlementBatchRepo {
 	// there can be multiple batches with the same name (excludes sequence number)
 	getBatchesByName(batchName: string): Promise<ISettlementBatch[]>;
```

Comment:
No need for ```getBatchesByNameWithPagi()```, please change getBatchesByName() to accept the optional pagination parameters